content/deserialize-instruction-data.md

```diff
@@ -12,18 +12,18 @@ objectives:
 
 # TL;DR
 
-- Karamihan sa mga programa ay sumusuporta sa **maraming discrete na mga tagubilin** - magpapasya ka kapag isinusulat mo ang iyong programa kung ano ang mga tagubiling ito at kung anong data ang dapat kasama sa kanila
-- Ang mga kalawang **enums** ay kadalasang ginagamit upang kumatawan sa mga hiwalay na tagubilin ng programa
+- Karamihan sa mga programa ay sumusuporta sa **multiple discrete instructions** - magpapasya ka kapag isinusulat mo ang iyong programa kung ano ang mga instructions ito at kung anong data ang dapat kasama sa kanila
+- Ang mga kalawang **enums** ay kadalasang ginagamit upang kumatawan sa mga hiwalay na instructions ng programa
```

Comment:
kalawang to `Rust` please also check if we still have several instance of `kalawang`

Reply:
Noted po, sir.